### PR TITLE
Withdraw stale cips

### PR DESCRIPTION
**Pull request by oed**
To be merged after #130 

This PR withdraws a number of CIPs that have gone stale and are no longer relevant.

### CODE REVIEW

**Review by AaronGoldman — COMMENTED** [review]
I think this looks good but we should make all of the cips have a revision date even if it is the same as the creation date.

**Comment by oed** [comment]
Sorry @AaronGoldman I now set the #130 branch as the base for this one. So some of your comments got stale here. Mind reviewing #130 separately?

**Review by stbrody — APPROVED** [review]
lgtm

File: CIPs/cip-1.md [hunk 1]
```diff
@@ -35,80 +33,63 @@ Here are some of the reasons why Ceramic needs a CIP process:
...
+> The Review status is an indication to the community that the CIP is ready for final review.
 
-An editor will reach out to provide the dates of the upcoming Ceramic Core Devs calls. Once you select one, your issue will be added to the agenda for that call where it will be discussed for inclusion in a future network upgrade. 
+Simply submit a new PR to update your CIP status to `Review`.
 
-If implementers agree to include your CIP in a future network upgrade, CIP editors will update your CIP to **Status: ACCEPTED**. Once your proposal has been released in a network upgrade, CIP editors will update your CIP to **Status: FINAL**.
+#### 4. Finalize your CIP
 
-If implementers decide not to include your CIP in a future network upgrade for whatever reason, you are always able to propose it again at a later time. It will remain as **Status: PENDING** in the CIP repo.
+> Only non-normative changes are allowed on CIPs in Final status.
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-1.md]: Seems like that should be covered in a new CIP if so?

File: CIPs/cip-120.md [hunk 2]
```diff
@@ -4,8 +4,7 @@ title: Multiple Pubsub Topics
 author: Joel Thorstensson <joel@3box.io>
 discussions-to: https://forum.ceramic.network/t/cip-120-multiple-pubsub-topics/
 status: Review
-category: Standards
-type: Core
+category: Core
 created: 2022-12-13
 edited: 2022-12-13
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-120.md]: I think that's not needed.

File: CIPs/cip-2.md [hunk 3]
```diff
@@ -5,10 +5,7 @@ author: Michael Sena (@michaelsena)
 discussions-to: https://github.com/ceramicnetwork/CIP/issues/39
 status: Final
 category: Meta
-type:
 created: 2020-06-14
```
[the remark this answers is not in the document]
Comment (oed) [reply CIPs/cip-2.md:8]: addressed in #130

**Pull request merged**